c_cpp/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN apt-get update && apt-get install -y --force-yes \
   && git clone https://github.com/google/googletest.git $HOME/googletest \
   && apt-get remove -y git \
   && apt-get autoremove -y \
+  && apt-get install -y wget \
```

Comment:
Put the boost install, and `wget` remove into this `RUN` command as well.

If you put the install and remove in separate `RUN` commands, a new layer is created and the image size doesn't get reduced by the removal.